content/en/synthetics/private_locations/configuration.md

```diff
@@ -114,7 +114,18 @@ Overrides the port for the private location status probes.
 `maxNbRedirects`
 : **Type**: Number <br>
 **Default**: `20`<br>
-Maximum number of redirections for HTTP and WebSocket tests. 
+Maximum number of redirections for HTTP and WebSocket tests.
+
+`variableOverride`
+: **Type**: String <br>
+Overrides the variables used in tests running on the Private Location. Format: `SECRET_VAR=value`.
+All variables imported this way will be obfuscated.
+
+`environmentVariableOverride`
+: **Type** String <br>
+Allows the usage of environment variables in the Private Location. It requires the environment variables to be imported in the containerized environment.
+Example for Docker: `docker run --env SECRET=value --environmentVariableOverride synthetics-private-location-worker`.
```

Comment:
```suggestion
Example for Docker: `docker run --env SECRET=value datadog/synthetics-private-location-worker --environmentVariableOverride`.
```

Reply:
btw should it be --env SECRET=value or --env SECRET ?

Reply:
Both work:

- `--env SECRET` will expose `SECRET=<value from the host>` in the container
- `--env SECRET=value` will expose `SECRET=value` in the container

Reply:
so option 2 = using `variableOverride`

Reply:
Yes